gam/kendall_tau_distance.py

```diff
@@ -147,10 +147,15 @@ def mergeSortDistance(r1, r2):
     return dist
 
 
-def pairwise_distance_matrix(rankings):
+def pairwise_distance_matrix(rankings, dask=False):
     from sklearn.metrics import pairwise_distances
-
-    D = pairwise_distances(rankings, metric=mergeSortDistance)
+    import numpy as np
+    from dask_ml.metrics.pairwise import pairwise_distances as dask_pairwise_distances
+    import dask.array as da
+    if dask:
+        D = dask_pairwise_distances(da.array(rankings), rankings, metric=mergeSortDistance)
```

Comment:
Mentioned this in `spearman_disntance`, that you should consider letting users pass in dask arrays rather than creating the dask arrays yourself here.